packages/playground/src/components/vm_deployment_table.vue

```diff
@@ -197,18 +203,20 @@ async function loadDeployments() {
   if (chunk2.count > 0 && migrateGateways) {
     await migrateModule(grid!.gateway);
   }
+  let chunk3: LoadedDeployments<any[]> = { count: 0, items: [], failedDeployments: [] };
```

Comment:
what are the items here?

Reply:
it is just an initial case, with empty values